### PR TITLE
remove chart studio from security program

### PR DESCRIPTION
**Pull request by BRONSOLO**
Resolves: https://github.com/plotly/dekn/issues/4537

FYI @alexcjohnson @x3rus @jingningzhang1 could I get a review of these changes? This was something that was discussed a few months back. Happy to provide more context as needed.

### CODE REVIEW

**Comment by BRONSOLO** [comment]
FYI I can also update any posts / program description we have in hackerone to reflect these changes.

File: _posts/fundamentals/2016-08-02-security.md [hunk 1]
```diff
@@ -6,8 +6,9 @@ title: Plotly Security & Vulnerability Program
 support: true
 ---
 
+**Update 05.04.2023:** We are no longer accepting security reports for Chart Studio. Please direct all security investigations to our Dash Enterprise product. A demo instance of this offering can be found here: https://sales-demo.plotly.com.
```
Comment (alexcjohnson) [inline _posts/fundamentals/2016-08-02-security.md:9]:
```suggestion
**Update May 4, 2023:** We are no longer accepting security reports for Chart Studio. Please direct all security investigations to our Dash Enterprise product. A demo instance of this offering can be found here: https://sales-demo.plotly.com.
```
Just to remove ambiguity between May 4 and April 5 😉 

Do we want to add anything about security in our OSS packages? We certainly accept and have paid for reports about XSS and related attacks in Dash and the graphing libraries but I don't know if this is explicitly reported anywhere.

Comment (BRONSOLO) [reply _posts/fundamentals/2016-08-02-security.md:9]: Thanks : ) I agree let's add something about OSS packages. I'll make that change.

**Comment by x3rus** [comment]
I agree !!! it's time consuming

**Comment by BRONSOLO** [comment]
Thanks @jingningzhang1 -- yes https://github.com/plotly/dekn/issues/4537 is the ticket I'm looking to resolve! I'll update the top comment.

**Comment by BRONSOLO** [comment]
@jingningzhang1 I believe @alexcjohnson was simply fixing how I specified the date. I was the one who was making the suggestion of including a link to https://sales-demo.plotly.com. 

>do we need to share the demo link?

I'm thinking it would be good to share the link to a production-level instance of DE so that folks can actually do meaningful investigations into the security as part of our program. 

>Should it go to an app that does not have authentication instead?

I think having this service behind an auth wall is best. No need to have users look to create apps and such. It does impose limitations to what vulnerabilities will actually surface but I don't think we want to go opening up a public DE just yet : )

**Comment by alexcjohnson** [comment]
> I don't think we want to go opening up a public DE just yet : )

Seems reasonable - at some point we should have one available and perhaps invite specific hackers to use it, but for now we can let it be a complete black box.

**Comment by BRONSOLO** [comment]
😅 finally got a moment to update this text. @alexcjohnson @jingningzhang1 any suggestions?

**Pull request merged**